scene/gui/split_container.cpp

```diff
@@ -39,7 +39,7 @@ void SplitContainerDragger::gui_input(const Ref<InputEvent> &p_event) {
 
 	SplitContainer *sc = Object::cast_to<SplitContainer>(get_parent());
 
-	if (sc->collapsed || !sc->get_containable_child(0) || !sc->get_containable_child(1) || sc->dragger_visibility != SplitContainer::DRAGGER_VISIBLE) {
+	if (sc->collapsed || !sc->get_containable_child(1) || sc->dragger_visibility != SplitContainer::DRAGGER_VISIBLE) {
```

Comment:
> sc->get_containable_child(1)

This seems like it is still hardcoded for two children.

Reply:
This actually checks for at least two children. When there is 0 or 1 children, `get_containable_child(1)` will be false and when there is 2 or more it will be true.
I changed this in a few places, but `get_containable_child(1)` implies `get_containable_child(0)`.

Reply:
Ah ok that makes sense. In such case, I think it would be good to write a comment. Or maybe be explicit about it by defining a variable instead:
```
bool has_at_least_two_containable_child = sc->get_containable_child(1);
if (sc->collapsed || has_at_least_two_containable_child || sc->dragger_visibility != SplitContainer::DRAGGER_VISIBLE) {
 ...
```
Something like that I'd say.